<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>herschelsDialogForm</class>
 <widget class="QWidget" name="herschelsDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>661</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="BarFrame" name="TitleBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="leftSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="stelWindowTitle">
        <property name="text">
         <string>Herschels</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="rightSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeStelWindow">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="Contents">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="Tabs">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="usesScrollButtons">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="sweepsTab">
         <attribute name="title">
          <string>Sweeps</string>
         </attribute>
         <layout class="QHBoxLayout" name="CCD_horizontalLayout_1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QFrame" name="sweepListFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="CCD_verticalLayout_1">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListView" name="sweepListView">
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="isWrapping" stdset="0">
                <bool>false</bool>
               </property>
               <property name="viewMode">
                <enum>QListView::ListMode</enum>
               </property>
               <property name="modelColumn">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="sweepsButtonFrame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="CCD_horizontalLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pushButtonMoveSweepUp">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="icon">
                   <iconset resource="../../../../data/gui/guiRes.qrc">
                    <normaloff>:/graphicGui/spinup.png</normaloff>:/graphicGui/spinup.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonMoveSweepDown">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="icon">
                   <iconset resource="../../../../data/gui/guiRes.qrc">
                    <normaloff>:/graphicGui/spindown.png</normaloff>:/graphicGui/spindown.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="SweepInfoFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="CCD_verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="SweepData">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="3" column="0">
                 <widget class="QLabel" name="TopDec_lbl">
                  <property name="text">
                   <string>Top Dec</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="SweepDate_lbl">
                  <property name="text">
                   <string>Date</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="EndRA_lbl">
                  <property name="text">
                   <string>Ending RA</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="BottomDec_lbl">
                  <property name="text">
                   <string>Bottom Dec</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="sweepName">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="baseSize">
                   <size>
                    <width>100</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="maxLength">
                   <number>50</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="StartRA_lbl">
                  <property name="text">
                   <string>Starting RA</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="SweepName_lbl">
                  <property name="text">
                   <string>Name:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="sweepStartRA"/>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="sweepEndRA"/>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="sweepStartDec"/>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="sweepEndDec"/>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="sweepDate">
                  <property name="toolTipDuration">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="submitSweep">
               <property name="text">
                <string>Submit Changes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pointToSweep">
               <property name="text">
                <string>Point To Sweep</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="projectSweep">
               <property name="text">
                <string>Project Sweeps</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="filterSweeps">
               <property name="text">
                <string>Show only current month's sweeps</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="sweepEffect">
               <property name="text">
                <string>Show Sweep Box</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="originalImageCheck">
               <property name="text">
                <string>Show Original Image</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="SweepverticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>86</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Set Time Rate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="timeRate">
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="enableOcularView">
               <property name="text">
                <string>Enable Ocular</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="playSweep">
               <property name="text">
                <string>Play Sweep</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stopSweep">
               <property name="text">
                <string>Stop Sweep</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="movieMode">
               <property name="text">
                <string>Play All Sweeps</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="sweepCalendar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <attribute name="title">
          <string>Calendar</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCalendarWidget" name="calendarWidget"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="openNewCalendar">
            <property name="text">
             <string>Open in New Dialog</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sweepListView</tabstop>
  <tabstop>pushButtonMoveSweepUp</tabstop>
  <tabstop>pushButtonMoveSweepDown</tabstop>
  <tabstop>sweepName</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../data/gui/guiRes.qrc"/>
 </resources>
 <connections/>
</ui>
